### DISCUSSION TITLE
Erigon debug and trace not working

### DISCUSSION THREAD

**Discussion by pikansj0s** [erigontech/erigon#5919] [Q&A]
Cannot do debug_traceCall or trace_call on my erigon eth node.

curl -H "Content-Type: application/json" -d '{"id": 1, "method": "debug_traceCall", "params": ["0xE486B7", {"tracer": "callTracer"}]}' http://172.16.20.21:8545/eth?apikey=123

{"jsonrpc":"2.0","id":1,"error":{"code":-32601,"message":"the method debug_traceCall does not exist/is not available"}}

What can be the problem? 

I have startet the node with "http.api=eth,debug,net,trace,web3,erigon,txpool,db,personal

**Comment by AskAlexSharov** [comment]
closing as too old
